### DISCUSSION TITLE
【任务】官网前端更新1｜20,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#139] [任务 | Bounty]
官网 [WTF.Academy](https://wtf.academy) 上线了 https://github.com/AmazingAng/WTFSolidity/discussions/134 。

但是我们可以看到它目前非常简陋，需要去迭代，让它变得更好。因此，我们发布了一些赏金任务，欢迎贡献者认领。

前端repo: https://github.com/WTFAcademy/frontend

## 任务1：贡献者荣誉墙
- 任务描述：写一个panel，滚动展示贡献者的头像、昵称。并且点击昵称跳转到他/她的github个人页。在最右端加一个expand按钮，可以展开看到所有贡献者。
- 赏金：5000 $PEOPLE
- 认领情况： @buttonwild 
<img width="717" alt="截屏2022-07-28 下午12 22 45" src="https://user-images.githubusercontent.com/14728591/181420059-a4f2b7fc-352c-4a2b-8d7e-792b8c81cec9.png">

## 任务2：官网主页UI设计
- 任务描述：让WTF学院主页变的更好看，但仍保持极简风格。除了目前有的元素，还需要添加贡献者荣誉墙，新闻栏的设计。
- 赏金：5000 $PEOPLE
- 认领情况： @KM-LEI
- 
## 任务3：github登陆模块
- 任务描述：在WTF主页上增加github登陆模块，允许用户利用github登陆。可以参照vercel.app的登陆模块
- 赏金：5000 $PEOPLE
- 认领情况：[vecpeng](https://github.com/vecpeng) 认领
- 完成情况：**已完成** Excellent 
- https://github.com/WTFAcademy/frontend/pull/1
- https://github.com/WTFAcademy/frontend/pull/2
- https://github.com/WTFAcademy/frontend/pull/3
- https://github.com/WTFAcademy/frontend/pull/4
- 技术文档：https://github.com/WTFAcademy/WTFTasks/blob/main/frontend/01_GithubOauth/readme.md

## 任务4：社区新闻/赏金任务发布栏
- 任务描述：写一个新闻/赏金任务栏，将发表在Github论坛上的WTF社区新闻和赏金任务同步到官网上。
- 赏金：5000 $PEOPLE
- 认领情况： @reborn-sama

**Comment by vecpeng** [comment]
想认领任务3，可以吗

**Reply by AmazingAng** [reply]
没问题

**Comment by reborn-sama** [comment]
4

**Comment by buttonwild** [comment]
1

**Comment by boombb12138** [comment]
hi~请问已经被认领的任务是不是不能重复认领？

**Reply by AmazingAng** [reply]
你想领哪个？可以组队做

**Comment by tvrrdfn** [comment]
1

**Comment by 4O4MAGlC1AN** [comment]
1
